docs/versioned/install/installing-contour.md

```diff
@@ -0,0 +1,209 @@
+---
+audience: administrator
+components:
+  - serving
+function: how-to
+---
+
+# Installing Contour for Knative
+
+This page shows how to install Contour in three ways:
+
+- By using Contour’s example YAML.
+- By using the Helm chart for Contour.
+- By using the Contour gateway provisioner.
+
+It then shows how to deploy a sample workload and route traffic to it through Contour.
+
+This guidance uses all default settings. No additional configuration is required.
```

Comment:
I liked how the Kourier docs provided information about configuration options.  It may be that `net-contour` does not have any separate configuration options, but if so, we should say that the configuration is done natively through Contour configuration, and people should see the guides at https://projectcontour.io/